packages/sources/svelte/src/components/VtmnPopover.svelte

```diff
@@ -0,0 +1,49 @@
+<script context="module">
+  /** @enum */
+  export const POSITION = {
+    TOP_LEFT: 'top-left',
+    TOP: 'top',
+    TOP_RIGHT: 'top-right',
+    RIGHT: 'right',
+    BOTTOM_RIGHT: 'bottom-right',
+    BOTTOM: 'bottom',
+    BOTTOM_LEFT: 'bottom-left',
+    LEFT: 'left',
+  };
+
+  /** @type {POSITION} */
+  export const DEFAULT_POSITION = POSITION.BOTTOM;
```

Comment:
Idea from @jleveugle, an `auto` position by default. I will create an issue for this.